README.md

```diff
@@ -231,6 +231,12 @@ services:
     # ...
 ```
 
+# /v1/events-test route
+If you are using EventGate as a service, and if `test_events` is configured,
+a `GET /v1/events-test` route will be added. When requested, the `test_events` will be produced as if
```

Comment:
1. Should it be a POST route requiring no body? having GET produce things doesn't seem safe...
2. should it be prefixed with `_` since it's an internal thingy? or, even not prefixed with `v1` but something else? like `_internal/v1/`
3. should it maybe be called in a more generic way? like `/_internal/v1/probe`?

Reply:
1. Has to be GET, I want to use this with a k8s readinessProbe which does not support POST.
2. & 3. Hm.  not sure. Let's discuss in IRC.

Reply:
Going with /v1/_test/events

Reply:
Love it!